pkg/grizzly/resources.go

```diff
@@ -30,6 +30,13 @@ func (ref ResourceRef) String() string {
 // Resource represents a single Resource destined for a single endpoint
 type Resource struct {
 	Body map[string]interface{}
+
+	Provenance struct {
+		Format     string
+		Location   string
```

Comment:
What's the difference between `Location` and `Path`? :confused:

Reply:
Heh, one is `FilePath` and the other is `LocationWithinFile`. Better names? Suggestions?